docs/content/get-started/build-workers.md

```diff
@@ -4,11 +4,11 @@ As you will be launching a cloud deployment, you need to build out the code exec
 
 In the Unity Editor, you first need to make sure Burst compilation is **disabled** from **Jobs** > **Enable Burst Compilation**. Then you can build your workers from the SpatialOS menu by clicking **Build for cloud** > **All workers**.
 
-<%(Callout type="info" message="Building workers for the first time may take while. Why not make yourself a cup of tea while you wait?")%>
+<%(Callout type="info" message="Building workers for the first time may take while (about 10 minutes). Why not make yourself a cup of tea while you wait? Or check out [our Youtube channel](https://www.youtube.com/channel/UC7BE8B2yUeQxPvZytk47NYw/videos)? Or both!")%>
 
 ![SpatialOS menu in Unity]({{assetRoot}}assets/unity-spatialos-menu.png)
 
-> **It has finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`
+> **It has finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`. (You may want to only display Info and Error messages in the Console.)
```

Comment:
maybe rephrase to "want to hide Warnings in the Unity Console" ? I think it's fine either way - depends on what you think is clearer for a user

Reply:
This doesn't really explain why you want to. I am guessing because there's a lot of noise in the Console window?
If so, "You may want to..." would be better as....

```suggestion
> **It has finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`. (As the build process displays lots of warnings in the Unity Console windows which you don't need to take action on, you could choose to set the Console window to display only info and error messages, so you can see the relevant messages.)
```

Reply:
Note to self - added to new PR